tests/random_test.py

```diff
@@ -1056,6 +1056,40 @@ def testOrthogonal(self, n, shape, dtype):
         atol=tol, rtol=tol,
       )
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_p={}_shape={}"\
+       .format(p, jtu.format_shape_dtype_string(shape, dtype)),
+       "p": p,
+       "shape": shape,
+       "dtype": dtype}
+      for p in [.5, 1., 1.5, 2., 2.5]
+      for shape in [(), (5,), (10, 5)]
+      for dtype in jtu.dtypes.floating))
+  def testGnormal(self, p, shape, dtype):
+    key = self.seed_prng(0)
+    samples = random.gnormal(key, p, shape, dtype)
```

Comment:
We should test with and without `jit` (see the `rand`, `crand` pattern used elsewhere in this file)